_data/tasks.yml

```diff
@@ -23,4 +23,6 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
+  - title: Using DNS Horizontal Autoscaling
```

Comment:
I agree that the title is too general--the rationale for that is that I wouldn't know what "Using DNS Horizontal Autoscaling" is for unless I already knew the feature existed and went looking for it specifically.

The title should explain the use case in one line, basically. What do you use DNS Horizontal Autoscaling for?